install/cupy_builder/install_build.py

```diff
@@ -23,8 +23,6 @@
 _rocm_path = 'NOT_INITIALIZED'
 _compiler_base_options = None
 
-use_hip = bool(int(os.environ.get('CUPY_INSTALL_USE_HIP', '0')))
```

Comment:
Just curious, this env var is no longer needed?

Reply:
It is present in [`context`](https://github.com/cupy/cupy/blob/master/install/cupy_builder/_context.py#L22-L23)

Reply:
The code was under refactoring and there were two flags for `CUPY_INSTALL_USE_HIP` (here and context).

Reply:
I see, I missed that!